crowbar_framework/app/views/repositories/index.html.haml

```diff
@@ -0,0 +1,80 @@
+.col-xs-12
+  .row
+    %h1.page-header
+      = t(".title")
+    .alert.alert-info
+      = t(".refresh_hint")
+
+  - repository_platforms.each_with_index do |platform, index|
+    .row
```

Comment:
the panel should be aligned with the header, as in the other panels. if i remove the `.row` it will not be aligned anymore